projects/core/src/modules/scrollable-host/scrollable-host.service.ts

```diff
@@ -14,10 +14,21 @@ export class SkyScrollableHostService {
     private windowRef: SkyAppWindowRef
   ) { }
 
-  public getScrollabeHost(elementRef: ElementRef): HTMLElement | Window {
```

Comment:
I acknowledge that this fix here for the typo in the function name is technically a breaking change. However, the typo was a pretty bad one and no one should be using this function outside of ourselves. So I felt ok going ahead and making the change. Happy to chat about it though.